server/initializers/checker-after-init.ts

```diff
@@ -153,6 +153,24 @@ function checkConfig () {
     }
   }
 
+  // Object storage
+  if (CONFIG.OBJECT_STORAGE.ENABLED === true) {
+    if (CONFIG.TRANSCODING.WEBTORRENT.ENABLED && !CONFIG.OBJECT_STORAGE.VIDEOS.BUCKET_NAME) {
```

Comment:
Transcoding can be updated at runtime so I think it would be simpler to force admins to fill all bucket names, even if they don't use webtorrent or hls videos